cocos/3d/CCSprite3D.h

```diff
@@ -200,6 +200,9 @@ class CC_DLL Sprite3D : public Node, public BlendProtocol
     */
     void setForce2DQueue(bool force2D);
 
+    Vector<Mesh*>& getMeshes() { return _meshes; }
```

Comment:
Here the return value is Vector& and _meshes is not const, so we can not add const declare. We can only change to Vector getMeshes() const, but in this way, it will return a copy _meshes, use more memory, please confirm how can we do.